.github/workflows/build-packages.yml

```
@@ -1821,7 +1837,9 @@ jobs:
       - name: Sign and upload cedarling_pg tarball
         working-directory: ${{ env.EXT_DIR }}
         env:
```

actionlint (1.7.12)

[error] expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)